server/api/ticketmaster.js

```diff
@@ -0,0 +1,24 @@
+const router = require('express').Router();
+const axios = require('axios');
+const dotenv = require('dotenv');
+dotenv.config();
+
+// GET /api/ticketmaster
+router.get('/', async (req, res) => {
+  try {
+    //TICKET MASTER
+    // const data = await axios.get(`https://app.ticketmaster.com/discovery/v2/events.json?countryCode=US&apikey=${process.env.TICKETMASTERAPIKEY}`)
+    // const events = await axios.get(`https://app.ticketmaster.com/discovery/v2/events.json?classificationName=music&countryCode=US&apikey=${process.env.TICKETMASTERAPIKEY}`)
+
+    const events = await axios.get(
+      `https://app.ticketmaster.com/discovery/v2/events.json?classificationName=music&countryCode=CA&apikey=${process.env.TICKETMASTERAPIKEY}`
+    );
+    console.log('full events object', events);
+    console.log('data', events.data._embedded.events);
+    res.send(events.data._embedded.events);
+  } catch (err) {
+    console.log(err);
```

Comment:
dont forget to next(err)
if there is an error, since you're not next()ing, it just hangs there forever